appveyor.yml

```diff
@@ -29,11 +29,11 @@ init:
   - git config --global core.autocrlf input
 
 cache:
-# Cache any downloads from sourceforge because it's really unreliable
+# Cache large downloads to avoid network unreliability
   - i686-4.9.2-release-win32-sjlj-rt_v4-rev3.7z
   - x86_64-4.9.2-release-win32-seh-rt_v4-rev3.7z
-  - llvm-3.7.1-i686-w64-mingw32-juliadeps-r04.7z
-  - llvm-3.7.1-x86_64-w64-mingw32-juliadeps-r04.7z
+  - llvm-3.7.1-i686-w64-mingw32-juliadeps-r09.7z
+  - llvm-3.7.1-x86_64-w64-mingw32-juliadeps-r09.7z
```

Comment:
Note, mostly to myself, that the separate llvm builds/downloads probably won't be necessary any more after this is merged and makes it into the binaries, at least we'll be able to get the dll's from the nightlies. Though we'd need to still get the (post-configure, I think) includes from somewhere, so maybe I'm coming around a bit on #16093.